cpp.hint

```diff
@@ -0,0 +1,5 @@
+// Hint files help the Visual Studio IDE interpret Visual C++ identifiers
```

Comment:
To be confirmed by the Mixxx Core Team, but I believe we are trying not to put any IDE specific config file in the repo. Perhaps you may add this in your [global git ignore](https://stackoverflow.com/a/7335487)?